test/audiovideocontroller/DefaultAudioVideoController.test.ts

```diff
@@ -339,6 +340,63 @@ describe('DefaultAudioVideoController', () => {
     await sendICEEventAndSubscribeAckFrame();
   }
 
+  const meetingStatusCheckExpectedEventNames = [
+    'meetingStartRequested',
+    'attendeePresenceReceived',
+    'meetingStartSucceeded',
+  ];
+
+  async function setupMeetingStatusCheck(
+    events: { name: EventName; attributes: EventAttributes }[]
+  ): Promise<{
+    stopCalled: boolean;
+    observer: AudioVideoObserver;
+    event_observer: EventObserver;
+  }> {
+    audioVideoController = new DefaultAudioVideoController(
+      configuration,
+      new NoOpDebugLogger(),
+      webSocketAdapter,
+      new NoOpMediaStreamBroker(),
+      reconnectController,
+      eventController
+    );
+    let stopCalled = false;
+    class TestObserver implements AudioVideoObserver {
+      audioVideoDidStop(): void {
+        stopCalled = true;
+      }
+    }
+    const observer = new TestObserver();
+    audioVideoController.addObserver(observer);
+    const event_observer = {
+      eventDidReceive(name: EventName, attributes: EventAttributes): void {
+        events.push({
+          name,
+          attributes,
+        });
+      },
+    };
+    eventController.addObserver(event_observer);
+    await start();
+    reconnectController.disableReconnect();
+    configuration.connectionTimeoutMs = 15000;
+    delay(new DOMMockBehavior().asyncWaitMs * 20).then(() => {
+      webSocketAdapter.send(makeLeaveAckFrame());
+    });
+    return { stopCalled, observer, event_observer };
```

Comment:
I'm not seeing the return object of `observer` and `event_observer` are used in each test verification. is it possible to maintain a global var and reset it to null after each test. 

nit: naming of event_observer